models/double_entry_transactions/int_quickbooks__refund_receipt_double_entry.sql

```diff
@@ -17,6 +21,76 @@ refund_receipt_lines as (
     from {{ ref('stg_quickbooks__refund_receipt_line') }}
 ),
 
+{% if using_refund_receipt_tax_line %}
+refund_receipt_tax_lines as (
+
+    select 
+        refund_receipt_id,
+        source_relation,
+        index + 10000 as index,
+        tax_rate_id,
+        amount,
+        tax_percent
+    from {{ ref('stg_quickbooks__refund_receipt_tax_line') }}
+),
+
+accounts as (
+
+    select *
+    from {{ ref('stg_quickbooks__account') }}
+),
+
+liability_accounts as (
+
+    select
+        account_id,
+        name,
+        source_relation
+    from accounts
+    where classification = 'Liability' 
+        and is_active
+),
+
+sales_tax_account as (
+
+    select
+        account_id,
+        source_relation
+    from accounts
+    where name = '{{ var('quickbooks__sales_tax_account_reference', 'Sales Tax Payable') }}'
+        and is_active
+),
+
+global_tax_account as (
+
+    select
+        account_id,
+        source_relation
+    from accounts
+    where name = '{{ var('quickbooks__global_tax_account_reference', 'Global Tax Payable') }}'
+        and is_active 
+),
+
+{% endif %}
+
+{% if using_tax_agency %}
```

Comment:
Same comment about trying to consolidate the conditionals into the parent variable if they're not used when that one is disabled.

Reply:
Consolidated.